### PR TITLE
Multiple tables

### PR DESCRIPTION
**Pull request by erickma1**
In this milestone, I did the following;

- Create a table named owners
- Create a table named species
- Modify table animals
- Insert data in owners, species table and update data in animals table

I also wrote the following queries to answer the following questions;

- What animals belong to Melody Pond?

1_animals_melody

- List of all animals that are pokemon (their type is Pokemon).

2_species_pokemon

- List all owners and their animals, remember to include those that don't own any animal.

3_owners

- How many animals are there per species?

4_species_count

- List all Digimon owned by Jennifer Orwell.

5_orwell

- List all animals owned by Dean Winchester that haven't tried to escape.

6_escape

- Who owns the most animals?

7_most_animals

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -15,4 +15,51 @@ INSERT INTO animals (name, date_of_birth, escape_attempts,neutered, weight_kg) V
...
+INSERT INTO species (name)
+VALUES
+   ('Pokemon'),
+   ('Digimon');
+
+UPDATE animals
+SET species_id = CASE
+    WHEN name LIKE '%mon' THEN (SELECT id FROM species WHERE name = 'Digimon')
+    ELSE (SELECT id FROM species WHERE name = 'Pokemon')
+  END;
```
Comment (Johnadibe) [inline data.sql:39]: > - [ ] Hello @erickma1, I noticed that you have not yet implemented the second requirement as stated [here](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_multiple_tables.md#:~:text=Modify%20your%20inserted,animals%20are%20Pokemon), which states that the species_id value must be included if all other animals are Pokemon. Please could you implement this requirement?👍

**Review by Johnadibe — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️
### Hi @erickma1👋🏽, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

mario_ladders

## Highlights
✅ You used GitFlow correctly 👍🏽
✅ Good commit history✔️
✅ Created a table named owners with the provided columns✔
✅ Created a table named species with the provided columns✔
✅ Inserted the provided data into the owners table✔
✅ Implemented all insertions and modifications in data.sql file✔
✅ Wrote queries using JOIN to answer the provided questions✔

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (erickma1) [reply data.sql:39]: Hi @Johnadibe , I am a little confused, I thought I had implemented that requirement as per the results in the query results attached below.

species_id

Comment (Johnadibe) [reply data.sql:39]: Hello @erickma1, Apologies, I quickly saw the Else statement you added to same line. Please you can resubmit for rereview okay👍

**Review by aamirkhan2478 — APPROVED** [review]
Hi @erickma1 , 

GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ You used Gitflow correctly :+1: 
✅ No linter error :+1: 
✅ Good commit messages :+1: 
✅ Good descriptive PR :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**